resteasy-core-spi/src/main/java/org/jboss/resteasy/resteasy_jaxrs/i18n/Messages.java

```diff
@@ -652,9 +652,8 @@
    @Message(id = BASE + 865, value = "Unable to determine base class from Type")
    String unableToDetermineBaseClass();
 
-   @Message(id = BASE
-         + 870, value = "Unable to extract parameter from http request: {0} value is \''{1}\'' for {2}", format = Format.MESSAGE_FORMAT)
-   String unableToExtractParameter(String paramSignature, String strVal, AccessibleObject target);
+   @Message(id = 3870, value = "Unable to extract parameter from http request: %s value is '%s'")
```

Comment:
We can make them explicit with the with positional arguments. Generally though it shouldn't be a problem unless some does some huge refactor.

I did run the tests which is something I want to look at because it breaks the checkstyle when you run them :) But yes, if I skip the checkstyle they pass.

Reply:
Ok, cool. I'll merge and port to 4.5 and 4.6.